assets/js/components/ExecutionResults/CheckResultOutline.test.jsx

```diff
@@ -43,11 +43,11 @@ describe('CheckResultOutline Component', () => {
     const checkID = faker.datatype.uuid();
     const clusterName = faker.lorem.word();
 
-    const expectationName1 = faker.company.name();
-    const expectationName2 = faker.color.human();
-    const expectationName3 = faker.color.human();
-    const expectSameExpectationName1 = faker.lorem.word();
-    const expectSameExpectationName2 = faker.lorem.word();
+    const expectationName1 = faker.datatype.uuid();
```

Comment:
@dottorblaster I applied the suggested change about uuids.
 
I am afraid, though, that the _should expectation names be  uuids?_ doubt might rise, which is not the case. 

As long as it clear that we just want strings that are unique in a restricted context and not necessarily uuids, we should be good.

Reply:
Yeah I agree, for me it's clear, I don't know if you want to leave a comment or something :sweat_smile: